CMakeLists.txt

```diff
@@ -326,7 +326,7 @@ if(USE_PROFILER)
   file(GLOB RUNTIME_GRAPH_DEBUG_SRCS src/runtime/graph/debug/*.cc)
```

Comment:
missed this

Reply:
I think this is fine, the var name derives from the path. or do you mean we should fix path to e.g. src/runtime/graph_executor/...?